src/App.js

```diff
@@ -6,11 +6,14 @@ import Header from "./Header/Header";
 import ItemList from "./List/ItemList";
 import getToken from "./lib/token";
 import Form from "./Form/Form";
+import * as firebase from "./lib/firebase";
 
 function App() {
   const [isListSelected, setIsListSelected] = useState(true);
   const [isAddSelected, setIsAddSelected] = useState(false);
   const [token, setToken] = useState("");
+  const [showNew, setShowNew] = useState(false);
+  const [showExisting, setShowExisting] = useState(false);
```

Comment:
So, I noticed for these two, you're using comments down below -- I think those comments might be extraneous if you named these better. ie. `showNewToken` & `showExistingToken`. Long variable names are fine as long as they are descriptive.

Reply:
Thanks for this suggestion. Naming things is hard. Haha this was actually the 2nd name we gave to these. I've just incorporated these changes.